xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-mainwiki/src/main/resources/WikiManager/CreateWiki.xml

```diff
@@ -866,6 +868,11 @@ $xwiki.getURL('XWiki.XWikiPreferences', 'admin', "editor=globaladmin&amp;section
         wikiName.value = prettyName.value.replace(/[^a-zA-Z0-9]/g,'_').toLowerCase();
         validateWikiName();
       }
+      // No matter the path, at this point, the wikiName has been already validated.
```

Comment:
This is not true, because the wiki name validation is **asynchronous**. We can have this case:

* the user types an **invalid** wiki name, which triggers a validation HTTP request
* the network is slow, so the user can type the wiki pretty name before the validation response is received; this enables the Next button => the user can move to the next step with an invalid wiki name